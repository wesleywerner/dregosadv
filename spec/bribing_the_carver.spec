superbrief
gonear tribe portal
purloin trap, drawing
abstract chicken to trap
show trap to man
give trap to man
examine carver
give drawing to carver
examine carver
wait
wait
wait
wait
wait
wait
examine carver
wait
wait
examine carver
look
take sun key
