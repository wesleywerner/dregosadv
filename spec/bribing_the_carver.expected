  THIS GAME IS IN DEBUG MODE
  
  Dr Ego and the Egg of Man-Toomba
  An Interactive Fiction Story
  by Special Agent.
  Release 1 / Serial number 400410 / Inform v6.33 Library v6.12.3pre SX
  
  The Maro River (in the canoe)
. 
  In the canoe I can also see a trap (which is closed and empty), a fishing
  spear and a fishing net.
  
> >  Dr Ego and the Egg of Man-Toomba is now in its "superbrief" mode, which
  always gives short descriptions of locations (even if you haven't been
  there before).
  
> >  Tribe
  
  A man sits nearby, carving a piece of wood.
  An unexplored path is to the west.
  
> >  trap: [Purloined.]
  drawing of the sun key: [Purloined.]
  
> >  [Abstracted.]
  
> >  He seems impressed!
  
> >  The carver has a big grin on his face, he removes the hen from the trap and
  takes her over to the firepit, handing the hen to somebody else. He returns
  and gestures to me for the drawing.
  
> >  The carver is waiting for the drawing of the sun key.
  
> >  I give him the drawing, picks a piece of wood from his pile and begins
  carving. This may take a while, I could return the trap to my guide while I
  wait.
  
  [I scored my first point! Type SCORE/FULL to list them.]
  
> >  The carver is busy working on my sun key replica.
  
> >  The Maro River
  
  My guide sits in the canoe gazing out over the moving river, the salty
  water lapping up the sides of his craft.
. 
> >  "Thanks friend, I hope you caught that chicken."
  
> >  Tribe
  
  A man sits nearby, carving a piece of wood.
  An unexplored path is to the west.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  
> >  Time passes.
  
> >  The carver is busy working on my sun key replica.
  
> >  Time passes.
  
> >  Time passes.
  The carver gestures to me that he is done with the carving.
  
> >  He is busy carving a rough piece of dry wood using a knife and other
  assorted hand tools. There are half a dozen carved wooden statuettes and
  figurines in various stages of completion on the ground beside him. His
  handiwork tells me he is very skilled in his trade.
  
> >  Tribe
  
  A man sits nearby, carving a piece of wood.
  
  I can also see a sun key here.
  An unexplored path is to the west.
  
> >  Taken.
  
> >